rita/src/rita_client/dashboard/eth_private_key.rs

```diff
@@ -0,0 +1,52 @@
+use super::*;
+use clarity::PrivateKey;
+
+pub fn get_eth_private_key(_req: HttpRequest) -> Box<dyn Future<Item = HttpResponse, Error = Error>> {
+    debug!("/eth_private_key GET hit");
+
+    let mut ret = HashMap::new();
+
+    match SETTING.get_payment().eth_private_key {
+        Some(pk) => {
+            ret.insert("eth_private_key".to_owned(), format!("{:x}", pk));
+        }
+        None => {
+            let error_msg = "No mesh IP configured yet";
+            warn!("{}", error_msg);
+            ret.insert("error".to_owned(), error_msg.to_owned());
+        }
+    }
+
+    Box::new(future::ok(HttpResponse::Ok().json(ret)))
+}
+
+pub fn set_eth_private_key(eth_private_key_data: Json<HashMap<String, String>>) -> Result<HttpResponse, Error> {
```

Comment:
Why post a hashmap object? so that we can set dozens of keys at once?

If you want a post format like {"eth_key": "actual key data I'm too lazy to type"} then you need an object like this

#[derive(Deserialize)]
struct KeyItem {
eth_key: PrivateKey
}

The Json fromat of the struct keyitem would then be exactly as above, you would derive deserialize so that the struct could be read from it's json format. 

The alternative is that you just have it take a Json<Privatekey>

which could get kinda strange but if it works right would be {"private key data I don't want to type"} in the post request. 

The finally option is a path variable which is how you get things like 

POST /endpoint/<actual key value goes here>

and that looks like Path<PrivateKey> or somthing like that. In theory we can use it here because no spaces, but it's also not something we want to screw up by involving it in strange url limitations so I don't think we should go this way. 

KeyItem is probably the way to go.